experimental/saiexperimentaldashtrustedvni.h

```diff
@@ -0,0 +1,376 @@
+ */
+typedef sai_status_t (*sai_get_global_trusted_vni_entry_attribute_fn)(
+        _In_ const sai_global_trusted_vni_entry_t *global_trusted_vni_entry,
+        _In_ uint32_t attr_count,
+        _Inout_ sai_attribute_t *attr_list);
+
+/**
+ * @brief Bulk create global trusted VNI entry
+ *
+ * @param[in] object_count Number of objects to create
+ * @param[in] global_trusted_vni_entry List of object to create
+ * @param[in] attr_count List of attr_count. Caller passes the number
+ *    of attribute for each object to create.
+ * @param[in] attr_list List of attributes for every object.
+ * @param[in] mode Bulk operation error handling mode.
+ * @param[out] object_statuses List of status for every object. Caller needs to
+ * allocate the buffer
+ *
+ * @return #SAI_STATUS_SUCCESS on success when all objects are created or
+ * #SAI_STATUS_FAILURE when any of the objects fails to create. When there is
+ * failure, Caller is expected to go through the list of returned statuses to
+ * find out which fails and which succeeds.
+ */
+typedef sai_status_t (*sai_bulk_create_global_trusted_vni_entry_fn)(
+        _In_ uint32_t object_count,
+        _In_ const sai_global_trusted_vni_entry_t *global_trusted_vni_entry,
+        _In_ const uint32_t *attr_count,
+        _In_ const sai_attribute_t **attr_list,
+        _In_ sai_bulk_op_error_mode_t mode,
+        _Out_ sai_status_t *object_statuses);
+
+/**
+ * @brief Bulk remove global trusted VNI entry
+ *
+ * @param[in] object_count Number of objects to remove
+ * @param[in] global_trusted_vni_entry List of objects to remove
+ * @param[in] mode Bulk operation error handling mode.
+ * @param[out] object_statuses List of status for every object. Caller needs to
+ * allocate the buffer
+ *
+ * @return #SAI_STATUS_SUCCESS on success when all objects are removed or
+ * #SAI_STATUS_FAILURE when any of the objects fails to remove. When there is
+ * failure, Caller is expected to go through the list of returned statuses to
+ * find out which fails and which succeeds.
+ */
+typedef sai_status_t (*sai_bulk_remove_global_trusted_vni_entry_fn)(
+        _In_ uint32_t object_count,
+        _In_ const sai_global_trusted_vni_entry_t *global_trusted_vni_entry,
+        _In_ sai_bulk_op_error_mode_t mode,
+        _Out_ sai_status_t *object_statuses);
+
+typedef struct _sai_dash_trusted_vni_api_t
+{
+    sai_create_eni_trusted_vni_entry_fn              create_eni_trusted_vni_entry;
+    sai_remove_eni_trusted_vni_entry_fn              remove_eni_trusted_vni_entry;
+    sai_set_eni_trusted_vni_entry_attribute_fn       set_eni_trusted_vni_entry_attribute;
+    sai_get_eni_trusted_vni_entry_attribute_fn       get_eni_trusted_vni_entry_attribute;
+    sai_bulk_create_eni_trusted_vni_entry_fn         create_eni_trusted_vni_entries;
+    sai_bulk_remove_eni_trusted_vni_entry_fn         remove_eni_trusted_vni_entries;
+
+    sai_create_global_trusted_vni_entry_fn           create_global_trusted_vni_entry;
```

Comment:
will be better to put global ones in the front.

Reply:
Fixed